README.md

```
@@ -16,141 +16,326 @@
-- **[Page interactions](https://docs.notte.cc/side/fullstack/page_interactions)** → observe website states and execute actions using intuitive natural language commands — granular control while maintaining the simplicity of natural language interaction
-- **[Secrets Vault](https://docs.notte.cc/side/fullstack/vault)** → enterprise-grade credential management for your Sessions & Agents
+API service:
+- **[Stealth Browser Sessions](https://docs.notte.cc/concepts/sessions)** → on-demand headless browser instances (Chrome, Firefox, Chromium), built in & custom proxy config, CDP compatible, cookie integration, session replay, automated captcha solving
+- **[Secrets Vaults](https://docs.notte.cc/concepts/vaults)** → enterprise-grade 
+credential management to securely store emails, passwords, MFA tokens, SSO, etc.
+- **[Personas](https://docs.notte.cc/concepts/personas)** → self-service digital identities with unique email addresses, phone numbers, 2FA handling for seamless account creation and authentication workflows
 
 # Quickstart
 
-We provide an effortless hosted API. To run the agent you'll need to sign up on the [Notte Console](https://console.notte.cc) and create a free Notte API key 🔑 We've prepared a small quickstart script for you! Just set your `NOTTE_API_KEY` as an environment variable and run:
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)